json-reporter.js

```diff
@@ -0,0 +1,30 @@
+class MyCustomReporter {
+  constructor(globalConfig, options) {
+    this._globalConfig = globalConfig;
+    this._options = options;
+  }
+
+  onRunComplete(contexts, results) {
+    results.testResults[0].testResults.forEach((item) => {
+      if (item.status !== "failed") {
+        return;
+      }
+      const newLine = "%0A";
```

Comment:
We have to url encode new lines in order to produce a matcher message that can spawn over multiple lines